.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
if using visual studio please look this over https://github.com/github/gitignore/blob/master/VisualStudio.gitignore